Core/Core/Courses/CourseListView.swift

```diff
@@ -41,33 +39,20 @@ public struct CourseListView: View {
     static var searchBarHeight: CGFloat = UISearchBar().sizeThatFits(.zero).height
 
     public init(allCourses: Store<GetAllCourses>? = nil) {
-        let loadingSubject = CurrentValueSubject<Bool, Never>(true)
-        if let allCourses = allCourses {
-            self.allCourses = allCourses
-            loadingSubject.send(false)
-        } else {
-            self.allCourses = AppEnvironment.shared.subscribe(GetAllCourses()) { store in
```

Comment:
Now the `store` parameter in the callback isn't needed anymore, I could go ahead and remove it again

Reply:
Went ahead and did it